Fix tests as reported by ${WRKSRC}/_build/meson-logs/testlog.txt

# Contents don't match expected contents
# --- /tmp/.XSRW02	2025-01-22 06:33:26.302100000 -0500
# +++ /tmp/.0ORW02	2025-01-22 06:33:26.302186000 -0500
# @@ -4,7 +4,7 @@
#  DejaVu Sans 20
#  Noto Sans CJK JP 20
#  Droid Sans Japanese 20
# -Cantarell 20
# +Cantarell 20 @wght=400
#  Amiri 20
#  DejaVu Sans Bold 20
#  Font Awesome 5 Free Solid Heavy 20
# 
not ok 1 /fontsets/mono2

--- tests/fontsets/mono2.orig	2025-01-18 22:41:03 UTC
+++ tests/fontsets/mono2
@@ -4,7 +4,7 @@ Droid Sans Japanese 20
 DejaVu Sans 20
 Noto Sans CJK JP 20
 Droid Sans Japanese 20
-Cantarell 20
+Cantarell 20 @wght=400
 Amiri 20
 DejaVu Sans Bold 20
 Font Awesome 5 Free Solid Heavy 20
